contracts/StablecoinConverter.sol

```diff
@@ -7,6 +7,11 @@ import "solidity-bytes-utils/contracts/BytesLib.sol";
 import "./libraries/TokenConservation.sol";
 
 
+/** @title Stable Coin Converter - A decentralized exchange for stable tokens as a batch auciton.
+ * @author Alex Herrmann - <alex@gnosis.pm>
+ * @author Benjamin Smith - <ben@gnosis.pm>
+ * @author Felix Leupold - <felix@gnosis.pm>
```

Comment:
not too keen on being personally mentioned here. Maybe we should have a dfusion team alias?

Reply:
Yes, sure... although this author convention has been used for all other gnosis contracts.